src/strings/PowerFxResources.en-US.resx

```diff
@@ -5453,4 +5453,8 @@
     <value>Table doesn't contain any column of text type.</value>
     <comment>Error Message.</comment>
   </data>
+  <data name="ErrDeprecatedDotUseShowColumn" xml:space="preserve">
+    <value>Deprecated use of '.'. Instead use ShowColumn().</value>
```

Comment:
>Instead use ShowColumn() [](http://example.com/codeflow?start=34&length=24)

Suggestion: 'Please use the 'ShowColumns' function instead'.

Reply:
updated, wasn't aware of this {Locked=} syntax. Thanks for pointing it out.